docs/sources/flow/setup/deploy-agent.md

```diff
@@ -0,0 +1,125 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/setup/deploy-agent/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/setup/deploy-agent/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/setup/deploy-agent/
+canonical: https://grafana.com/docs/agent/latest/flow/setup/start-agent/
+description: Learn about possible deployment topologies for Grafana Agent
+menuTitle: Deploy Grafana Agent
+title: Grafana Agent deployment topologies
+weight: 900
+---
+
+# Deploying Grafana Agent
+
+Grafana Agent is a flexible, vendor-neutral telemetry collector. This
+flexibility means that the Agent doesn’t enforce a specific deployment topology
+but can work in multiple scenarios.
+
+This page lists common topologies used for deployments of Grafana Agent, when
+to consider using each, issues that users may run into, as well as scaling
+considerations.
+
+## As a centralized collection service
+Deploying the Grafana Agent as a centralized service is recommended for
+collecting application telemetry. This allows a smaller set of agents to
+coordinate service discovery, collection, and remote writing.
+
+![](../../../assets/deploy-agent/centralized-collection.png)
```

Comment:
We have some guidelines for media that covers file name conventions and where media should be stored.

* File names: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#media-asset-file-naming-conventions
* Asset upload location: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#where-to-store-media-assets

This means that we should move these images out of `/docs/assets` before we merge.

Reply:
TIL, thanks for the heads up!